### ISSUE TITLE
Make graph node initial positions more deterministic

### ISSUE THREAD

**Issue by maximusunc** [NCATS-Gamma/robokop-rewrite#193]
I wonder if it would make the subsequent simulation easier if we start with quasi-random positions rather than uniformly random ones. Something like a Halton sequence: https://en.wikipedia.org/wiki/Halton_sequence

This would also mean that the positions are in fact deterministic, which could be nice.

_Originally posted by @patrickkwang in https://github.com/NCATS-Gamma/robokop-rewrite/pull/187#discussion_r646642733_

**Comment by maximusunc** [comment]
I don't think this will save any computation expenses as I think the simulation will still run through all its 'ticks' (the nodes just won't move as much). It might help us position nodes more logically for the query graph, but that doesn't seem necessary to me.

**Comment by patrickkwang** [comment]
Less randomness was not the original motivation here, just a potential side benefit. The main benefit of a low-discrepancy sequence, hopefully, would be that the nodes have to move less to arrive at steady-state positions. For large knowledge graphs right now we often terminate the simulation before it reaches this steady state. If it works well, we could even shorten the simulation.
